### ISSUE TITLE
Update to WiFiman 1.1.0 package

### ISSUE THREAD

**Issue by J0hnny007** [abn/wifiman-desktop-rpm#1]
Hello,

Is there a plan/possibility to update the RPM to use the new 1.1.0 WiFiman release?
I already tried converting the .deb with alien but ran into a system library conflict.

**Comment by abn** [comment]
I can take a look, but looks like it will require a bit of effort as they seem to have changed a lot of things in the packaging and also seems to have dropped arm support.

**Comment by gbcox** [comment]
> Hello,
> 
> Is there a plan/possibility to update the RPM to use the new 1.1.0 WiFiman release? I already tried converting the .deb with alien but ran into a system library conflict.

Yes, I had the same issue.  I created a makeshift solution to get around the issue until there is time to create a rpm for the new release.  It is x86_64 only: https://tso.bzb.us/2024/12/using-unifi-wifiman-desktop-with-fedora.html
